app/elements/batch-image-search/batch-image-search.html

```diff
@@ -0,0 +1,379 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/button-action/button-action.html">
+<link rel="import" href="../../bower_components/file-upload/file-upload.html">
+<link rel="import" href="../../bower_components/paper-dialog/paper-dialog.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../../bower_components/paper-input/paper-textarea.html">
+<link rel="import" href="../../bower_components/image-thumbnail-with-link/image-thumbnail-with-link.html">
+<link rel="import" href="../../bower_components/image-thumbnail/image-thumbnail.html">
+<link rel="import" href="../../bower_components/loading-spinner/loading-spinner.html">
+
+<dom-module id="batch-image-search">
```

Comment:
The reason why "batch phone" search was named that way was because the input was a batch of phone numbers.  Since this only takes a single image as input, I think it would make more sense to simply call it image-search (or similar-image-search).